client/src/Features/AssignmentLinks/AssignmentLinkEditor.tsx

```diff
@@ -18,6 +18,24 @@ interface AssignmentLinkEditorProps {
 }
 export type AssignmentLinkEditorStyles = SimpleComponentStyles<'root'>;
 
+const LinkTextValidation = (value: string | undefined, MAXLEN: number) => {
+  if(value && value?.length>MAXLEN){

```

Comment:
The following might be enough?
```suggestion
  if(value?.length > MAXLEN){
```

Reply:
It actually gives an error, I tried this on my end!